core/src/main/java/org/apache/calcite/tools/RelBuilder.java

```diff
@@ -2486,6 +2490,11 @@ private RelBuilder aggregate_(GroupKeyImpl groupKey,
         return rewriteAggregateWithDuplicateGroupSets(groupSet, groupSetMultiset,
             aggCalls);
       }
+      Optional<RelTrait> pivotRelTrait = stack.getFirst().rel.getTraitSet().stream()
+          .filter(it -> it instanceof PivotRelTrait).findFirst();
```

Comment:
here it is no need of stream it can done simply like this:

```
Optional<RelTrait> pivotRelTrait = Optional.ofNullable(stack.getFirst().rel.getTraitSet().getTrait(PivotRelTraitDef.instance));
```

Reply:
resolved

Reply:
this is exceeding the line length. fix this